bouncer/src/repo/core/model/bson/repo_bson_factory.h

```diff
@@ -265,12 +267,13 @@ namespace repo {
 				* @return returns a mesh node
 				*/
 				static MeshNode makeMeshNode(
```

Comment:
2 thoughts here:

1. I think we should separate MeshNode and SuperMesh node. They could converge to a single private function in the cpp of this file, but I think it'll give better readability and less confusion about what this id array is suppose to represent.
2. We should rename `ids` to subMeshIds` or something. as it definitely confused me when I read this code 😆

Reply:
Done!